### DISCUSSION TITLE
XRemotePreference and Framework API

### DISCUSSION THREAD

**Discussion by ghost** [LSPosed/LSPosed#899] [Q&A] [answered]
Until now we use Xposed Framework API released from rovo89, but now with XRemotePreference which framework will use?
I'd like to test XRemotePreference.

**Comment by yujincheng08** [comment, marked as answer]
Dreamland will also use it.

**Reply by ghost** [reply]
Sorry I don't understand, what is Dreamview?

**Reply by ghost** [reply]
I never tried dreamland, but my question is: how to use on a project the new library which include XRemotePreference?
I must generate a my jar which include the new xposed framework api or there will be a new maven repository?

**Reply by yujincheng08** [reply]
will be. but its not yet ready

**Reply by Huper7777** [reply]
developer@tele2.ru
